src/sonic_ax_impl/mibs/ieee802_1ab.py

```diff
@@ -511,26 +511,30 @@ def update_rem_if_mgmt(self, if_oid, if_name):
                 return
             time_mark = int(lldp_kvs[b'lldp_rem_time_mark'])
             remote_index = int(lldp_kvs[b'lldp_rem_index'])
-            subtype = self.get_subtype(mgmt_ip_str)
-            ip_hex = self.get_ip_hex(mgmt_ip_str, subtype)
-            if subtype == ManAddrConst.man_addr_subtype_ipv4:
-                addr_subtype_sub_oid = 4
-                mgmt_ip_sub_oid = (addr_subtype_sub_oid, *[int(i) for i in mgmt_ip_str.split('.')])
-            elif subtype == ManAddrConst.man_addr_subtype_ipv6:
-                addr_subtype_sub_oid = 6
-                mgmt_ip_sub_oid = (addr_subtype_sub_oid, *[int(i, 16) if i else 0 for i in mgmt_ip_str.split(':')])
-            else:
-                logger.warning("Invalid management IP {}".format(mgmt_ip_str))
-                return
-            self.if_range.append((time_mark,
-                                  if_oid,
-                                  remote_index,
-                                  subtype,
-                                  *mgmt_ip_sub_oid))
-
-            self.mgmt_ips.update({if_name: {"ip_str": mgmt_ip_str,
-                                            "addr_subtype": subtype,
-                                            "addr_hex": ip_hex}})
+
+            for mgmt_ip in mgmt_ip_str.split(','):
+                ipa  = ipaddress.ip_address(mgmt_ip)
+                subtype = self.get_subtype(mgmt_ip)
+                ip_hex = self.get_ip_hex(mgmt_ip, subtype)
+
+                if subtype == ManAddrConst.man_addr_subtype_ipv4:
+                    addr_subtype_sub_oid = 4
+                    mgmt_ip_sub_oid = (addr_subtype_sub_oid,) + tuple(i for i in ipa.packed)
+                elif subtype == ManAddrConst.man_addr_subtype_ipv6:
+
+                    addr_subtype_sub_oid = 16
```

Comment:
>16 [](start = 43, length = 2)

Could you double check whether it is 6?

Reply:
Sure. I checked other OS like cisco and did the snmpwalk for the same oid i.e. `1.0.8802.1.1.2.1.4.2`. It's 16. 

```
OBJECT  peerAddressType
     SYNTAX  InetAddressType { ipv4(1), ipv6(2) }
     DESCRIPTION
         "An implementation is only required to support IPv4
          and IPv6 addresses."

     OBJECT  peerAddress
     SYNTAX  InetAddress (SIZE(4|16))
     DESCRIPTION
         "An implementation is only required to support IPv4
          and globally unique IPv6 addresses."
```